scripts/apollo_perf.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+set -e
+
+TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${TOP_DIR}/scripts/apollo.bashrc"
+source "${TOP_DIR}/scripts/apollo_base.sh"
+
+function main() {
+    if ! command -v perf >/dev/null 2>&1; then
+      bash  "${TOP_DIR}/docker/scripts/install_perf.sh"
+      
+    fi
+    cd /apollo-simulator
+    bash build.sh run >/dev/null 2>&1 & 
+    sleep 1
+    ps -ef | grep "replay_engine" | grep -v grep | awk '{print $2}' |  xargs sudo perf record  -g -p
```

Comment:
`sudo perf record -g ./build.sh run` has the same function

Reply:
I have tried this way ,the problem:  libc10 shared library can't load  will happen. so use this way.